pkg/proto/nbcontract/v1/tlsbootstrappingconfig.proto

```diff
@@ -1,12 +1,9 @@
 syntax = "proto3";
 package nbcontract.v1;
 import "pkg/proto/nbcontract/v1/featurestate.proto";
-
+ 
 message TLSBootstrappingConfig {
-    // discuss whether this should be featureState or bool
-    // understand how defaulting works for enum, because we want default to be false
-    FeatureState tls_bootstrapping_status = 1; // need to review the relationship between tls_bootstrapping_status and secure_tls_bootstrapping_status. For example, if they are multually exclusive, then we should use oneof
-    FeatureState secure_tls_bootstrapping_status = 2;
-    string tls_bootstrap_token = 3; // Only required until Secure TLS bootstrapping in place. Would use kubelet identity after that.
-    string custom_secure_tls_bootstrap_appserver_appid = 4; // will follow up with the feature owner to see if this should be optional
+    bool enable_secure_tls_bootstrapping = 1;
+    optional string tls_bootstrap_token = 2; // Only required until Secure TLS bootstrapping in place. Would use kubelet identity after that.
```

Comment:
optional since it is only used when tls bootstrapping is enabled

Reply:
Removed optional, since no need to distinguish between unspecified and default value